test/wdio/lifecycle-async/cmp-a.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { h, Fragment } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('lifecycle-async', function () {
+  beforeEach(() => {
+    render({
+      template: () => <lifecycle-async-a></lifecycle-async-a>,
+    });
+  });
+
+  it('fire load methods in order', async () => {
+    await $('lifecycle-async-a').waitForExist();
+
+    await browser.pause(500);
```

Comment:
unfortunately without these `borwser.pause` calls here things aren't in place by the time we do all the `toHaveText` checks